Actors/ActorCounter/Scripts/upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+azure servicefabric application package copy CounterActorApplication fabric:ImageStore > /dev/null 2>&1
```

Comment:
So what's the experience now if someone wants to deploy / undeploy using cmdline? I see the install scripts have moved inside the scripts folder. Will they work as the copy command will now look for Application folder inside the script folder. Isnt it?